### PR TITLE
Added index.md documentation with all troubleshoot links

### PR DESCRIPTION
**Pull request by mantoanibroadwing**
Fixed links, and completed index.md

### CODE REVIEW

File: troubleshooting/index.md [hunk 1]
```diff
@@ -2,105 +2,78 @@
...
-
- * Kubernetes troubleshooting
-    * [General Kubernetes Commands for Troubleshooting](./kubernetes/Kubernetes_Troubleshooting_Information.md)
-    * [Kubernetes Log File Locations](./kubernetes/Kubernetes_Log_File_Locations.md)
-    * [Troubleshoot Liveliness or Readiness Probe Failures](./kubernetes/Troubleshoot_Liveliness_Readiness_Probe_Failures.md)
-    * [Troubleshoot Unresponsive kubectl Commands](./kubernetes/Troubleshoot_Unresponsive_kubectl_Commands.md)
-    * [Troubleshoot Kubernetes Node NotReady](./kubernetes/Troubleshoot_Kubernetes_Node_NotReady.md)
-
+    * [Hardware Discovery](known_issues/Hardware_Discovery.md)
+    * [SAT/HSM/CAPMC Component Power State Mismatch](https://github.com/Cray-HPE/docs-csm/blob/CASMINS-4311/troubleshooting/known_issues/component_power_state_mismatch.md)
```
Comment (johren-hpe) [inline troubleshooting/index.md:8]: I don't think these should be linking to the CASMINS-4311 branch.

Comment (johren-hpe) [reply troubleshooting/index.md:8]: I think these should all be relative paths so they stay with the content in the same branch.

**Review by johren-hpe — CHANGES_REQUESTED** [review]

File: troubleshooting/index.md [hunk 2]
```diff
@@ -2,105 +2,78 @@
...
 <a name="known-issues"></a>
 
-## Known Issues
-Listing of known issues and procedures to workaround them in this CSM release.
-
-<a name="known-issues-hardware-discovery"></a>
-
-### Hardware Discovery
-Known issues related to hardware discovery in a system.
- * [HMS Discovery job not creating RedfishEndpoints in Hardware State Manager](known_issues/discovery_job_not_creating_redfish_endpoints.md)
```
Comment (johren-hpe) [inline troubleshooting/index.md:28]: I don't see this in the list above.  Are you intentionally removing this known issue?

File: troubleshooting/index.md [hunk 3]
```diff
@@ -2,105 +2,78 @@
...
-
-<a name="initrd-not-found"></a>
-### `initrd.img.xz` Not Found
-
-This is a problem that is fixed in CSM 1.0+, but if your system was upgraded from CSM 0.9 you may run into this. Below is the full error seen when attempting to boot:
```
Comment (johren-hpe) [inline troubleshooting/index.md:33]: Same with this one.  Are you removing this because it is expected to be fixed in 1.2?

**Review by johren-hpe — CHANGES_REQUESTED** [review]

File: troubleshooting/index.md (hunk 2 above)
Comment (mantoanibroadwing) [reply troubleshooting/index.md:28]: This should be there, I added it again

File: troubleshooting/index.md (hunk 3 above)
Comment (mantoanibroadwing) [reply troubleshooting/index.md:33]: Added it back but as a link to document and not in root document

File: troubleshooting/index.md (hunk 1 above)
Comment (mantoanibroadwing) [reply troubleshooting/index.md:8]: Fixed

File: troubleshooting/index.md (hunk 3 above)
Comment (mantoanibroadwing) [reply troubleshooting/index.md:33]: Added again as link for a new document instead of content on same page

**Comment by johren-hpe** [comment]
Most of the links are still absolute paths.  These all should be relative paths.

There is also still a failure in the markdown-link-check.

**Pull request merged**